build/lint.mk

```diff
@@ -20,10 +22,11 @@ GOTOOLS += github.com/client9/misspell/cmd/misspell \
            github.com/gordonklaus/ineffassign \
            github.com/timakin/bodyclose \
            golang.org/x/lint/golint \
-           github.com/llorllale/go-gitlint/cmd/go-gitlint
+           github.com/llorllale/go-gitlint/cmd/go-gitlint \
+           github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
We should be able to remove all the specific linters and their Make targets now that this is working again.

Reply:
Also need to add golangci-lint to `/tools.go` to get it into go.mod.